modules/runner/src/main/ts/api/Main.ts

```diff
@@ -351,6 +127,13 @@ const loadtests = (data: HarnessResponse): void => {
 };
 
 const testrunner = (): void => {
+  const keepAliveTimer = setInterval(() => {
+    callbacks.sendKeepAlive(params.session, noop, () => {
+      // if the server shuts down stop trying to send messages
+      clearInterval(keepAliveTimer);
+    });
+  }, 5000);
```

Comment:
The 5000 should probably be a constant or given some kind of description.

Reply:
e.g. Use 5 seconds as a keep-alive heartbeat. Arbitrary chosen as 5. Or chosen to be 5 because some other value was x and y

Reply:
Done